tools/accuracy_checker/accuracy_checker/representation/segmentation_representation.py

```diff
@@ -17,6 +17,8 @@
 from enum import Enum
 from pathlib import Path
 from copy import deepcopy
+from skimage.measure import find_contours
+from shapely.geometry import Polygon
```

Comment:
it is not AC core requirements, it means that user may not have these libs installed, please make them optional.
BTW, regarding contours finding, the same functionality provided in opencv, probably, it will be better if you will use it (because opencv provided in the same package and do not require specific dependencies.
Also, one of the posible way of this function usage is writing to json file or providing via API to UI (written on another programming language), probably just return points for build building polygon contours will be enough.